drivers/adc/adc_nrfx_saadc.c

```diff
@@ -7,7 +7,8 @@
 #define ADC_CONTEXT_USES_KERNEL_TIMER
 #include "adc_context.h"
 #include <haly/nrfy_saadc.h>
-#include <zephyr/dt-bindings/adc/nrf-adc.h>
+#include <zephyr/dt-bindings/adc/nrf-saadc-v3.h>
+#include <zephyr/dt-bindings/adc/nrf-saadc-nrf54l.h>
```

Comment:
This change should be done in the first commit ("dts: nordic: refactor bindings helper symbols for SAADC").

Reply:
done